src/applications/edu-benefits/10282/config/form.js

```diff
@@ -1,144 +1,195 @@
   savedFormMessages: {
     notFound: 'Please start over to apply for 	education benefits.',
     noAuth:
       'Please sign in again to continue your application for 	education benefits.',
   },
-  title: 'Complex Form',
+
+  title: 'Apply for the IBM SkillsBuild Program',
+  subTitle:
+    'IBM SkillsBuild Training Program Intake Application (VA Form 22-10282)',
   footerContent: FormFooter,
   getHelp: FormHelp,
   defaultDefinitions: {
     fullName,
-    ssn,
-    date,
-    dateRange,
     usaPhone,
+    email,
   },
+  transformForSubmit: transform,
   chapters: {
-    applicantInformationChapter: {
-      title: 'Applicant Information',
+    personalInformation: {
+      title: 'Your personal information',
       pages: {
-        applicantInformation: {
-          path: 'applicant-information',
-          title: 'Applicant Information',
+        applicantName: applicantInformationName.applicantInformationField(),
+        veteranDesc: {
+          title: 'Your personal information',
+          path: 'applicant-information-1',
+          uiSchema: veteranDesc.uiSchema,
+          schema: veteranDesc.schema,
+        },
+        contactInfo: {
+          title: 'Your personal information',
+          path: 'applicant-information-2',
+          uiSchema: applicantContactInfo.uiSchema,
+          schema: applicantContactInfo.schema,
+        },
+        applicantCountry: {
+          title: 'Your personal information',
+          path: 'applicant-information-3',
+          uiSchema: applicantInformationCountry.uiSchema,
+          schema: applicantInformationCountry.schema,
+        },
+        applicantState: {
+          title: 'Your personal information',
+          path: 'applicant-information-4',
+          uiSchema: applicantState.uiSchema,
+          schema: applicantState.schema,
+          depends: formData => {
+            return formData.country === 'United States';
+          },
+        },
+        genderRaceQuestion: {
+          title: 'Your personal information',
+          path: 'applicant-information-5',
+          uiSchema: genderRaceQuestion.uiSchema,
+          schema: genderRaceQuestion.schema,
+        },
+        applicantRaceAndEthnicity: {
+          title: 'Your personal information',
+          path: 'applicant-information-6',
+          uiSchema: applicantRaceAndEthnicity.uiSchema,
+          schema: applicantRaceAndEthnicity.schema,
+          depends: formData => formData.raceAndGender === 'Yes',
+        },
+        applicantGender: {
+          title: 'Your personal information',
+          path: 'applicant-information-7',
+          uiSchema: applicantGender.uiSchema,
+          schema: applicantGender.schema,
+          depends: formData => formData.raceAndGender === 'Yes',
+        },
+      },
+    },
+    educationAndEmploymentHistory: {
+      title: ' Your education and employment history',
+      pages: {
+        highestLevelOfEducation: {
+          title: 'Your education and employment history',
```

Comment:
On mobile, the text should be styled to be left aligned only up to the step number.

![image](https://github.com/user-attachments/assets/94604b80-49e6-4051-96ee-345b239bfbd7)

Reply:
other forms have the same thing. it's not aligned